doc/source/ray-air/doc_code/tf_starter.py

```diff
@@ -52,18 +52,25 @@ def train_func(config: dict):
 
     dataset = session.get_dataset_shard("train")
 
+    def to_tf_dataset(dataset, batch_size):
+        def to_tensor_iterator():
+            for batch in dataset.iter_tf_batches(
+                batch_size=batch_size, dtypes=tf.float32
+            ):
+                yield tf.expand_dims(batch["x"], 1), batch["y"]
+
+        output_signature = (
+            tf.TensorSpec(shape=(None, 1), dtype=tf.float32),
+            tf.TensorSpec(shape=(None), dtype=tf.float32),
+        )
```

Comment:
```suggestion
            for batch in dataset.iter_tf_batches(
                batch_size=batch_size, dtypes=tf.float32
            ):
                yield batch["x"], batch["y"]

        output_signature = (
            tf.TensorSpec(shape=(None, ), dtype=tf.float32),
            tf.TensorSpec(shape=(None), dtype=tf.float32),
        )
```

I'm a little rusty, but would this work since the model definition explicitly has this step?
```python
            # Add feature dimension, expanding (batch_size,) to (batch_size, 1).
            tf.keras.layers.Flatten(),
```

Reply:
I may have too little context to comment. anyone else has a better idea? @richardliaw

Reply:
I think in both cases it does the same thing. OK to just take Matt's suggestion as it seems to simplify the code abit.